chore: prepare 1.4.3 release and deprecate project (#675)
redhat-developer/intellij-tekton: README.md

[![License](https://img.shields.io/github/license/redhat-developer/intellij-tekton?style=for-the-badge)](https://github.com/redhat-developer/intellij-tekton/blob/main/LICENSE)

# IntelliJ Tekton

> ## NOTE: This project is no more maintained and marked as deprecated.

Reviewer: ```suggestion
## NOTE: This plugin is no longer maintained and is officially deprecated.
```
